ga4gh/datamodel/__init__.py

```diff
@@ -324,25 +324,6 @@ class PhenotypeAssociationCompoundId(CompoundId):
         ('phenotypeAssociationId', 2)]
 
 
-class PhenotypeAssociationSetCompoundId(CompoundId):
```

Comment:
There are compound ID classes for the association and association set. Should there be entries for your other identifiers?